### ISSUE TITLE
no-array-callback-reference false positives with mongoose.Model

### ISSUE THREAD

**Issue by Samuel-Therrien-Beslogic** [sindresorhus/eslint-plugin-unicorn#1480]
mongoose Models have a `find` function to query a MongoDB Database. I think this could be added to `ignoredCallee`

```js
import { model, Schema, Types } from 'mongoose'

type Account = { _id: string, id: string }
const schema = new Schema<Account, Model<Account>, Account>()
const AccountModel = model('Account', schema)

AccountModel.find({_id: '000000000000000000000000'})
```
![image](https://user-images.githubusercontent.com/84519004/129114730-2d9092ab-42ef-4d57-a724-ce2c5f119fb7.png)

**Comment by fregante** [comment]
Related: 
- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1376

**Comment by XhmikosR** [comment]
We are also hitting this on Bootstrap due to https://github.com/twbs/bootstrap/blob/9852c9afbb4945e18cbb4836438c91894bedeee0/js/src/dom/selector-engine.js#L19.

I have disabled the rule globally but it would be awesome if this could be improved :)

**Comment by fregante** [comment]
Duplicate of https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1193
